### PR TITLE
Portfolio - mobile menu (Hamburger Menu)

### PR DESCRIPTION
**Pull request by katfogy**
**I created the Hamburger Menu with the following Features**
     -When the user clicks (or taps) the hamburger button, the mobile menu appears.
     -When the user clicks (or taps) the close (X) button, the mobile menu disappears.
     -When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
     -When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Booze33  👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
Please this is a pair-programming activity😨, Kindly make sure that your learning patterns have contributed to this branch and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -374,27 +404,38 @@ html {
     width: 100%;
     box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px;
     z-index: 6;
+    margin: 0 auto;
+  }
+
+  .info {
+    /* max-width: 1200px; */
```
Comment (divinecharlotte) [inline styles.css:411]: - [ ] kindly consider removing all the commented pice of code in your code base because they make your project look messy

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive PR ✔️ 




## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @katfogy 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**